marlowe-dashboard-client/src/Pickup/View.purs

```diff
@@ -226,14 +226,12 @@ renderPickupScreen :: forall p. Warn (Text "We need to add the Marlowe links.")
 renderPickupScreen state =
   div
     [ classNames [ "absolute", "top-0", "bottom-0", "left-0", "right-0", "overflow-auto", "z-0", "flex", "flex-col", "justify-between" ] ]
-    [ header
-        [ classNames [ "flex" ] ]
-        [ link "marlowe.io" "" ]
+    [ header_ []
     , pickupWalletScreen state
     , footer
-        [ classNames [ "flex", "justify-between" ] ]
-        [ link "Docs" ""
-        , link "Marketplace" ""
+        [ classNames [ "flex", "justify-between", "p-6", "lg:py-8", "md:px-5pc" ] ]
+        [ link "marlowe.finance" "https://marlowe.finance"
```

Comment:
I think we're going with `marlowe-finance.io` as the official link. Also we can remove the "We need to add the Marlowe links" warnings now.

Reply:
I thought that marlowe.finance was the one that we wanted but is still "transferring". Changing to marlowe-finance.io as we already have that